src/test/regress/expected/multi_insert_select.out

```diff
@@ -657,8 +657,7 @@ DEBUG:  Router planner cannot handle multi-shard select queries
 -- but prefer to prevent inlining of the CTE
 -- in order not to diverge from pg 11 vs pg 12
```

Comment:
minor: the above comment is `but prefer to prevent inlining of the CTE` not relevant anymore